.circleci/config.yml

```diff
@@ -897,6 +897,17 @@ jobs:
               echo "Skipping doc deploy (not on master)."
             fi
 
+  check-yellow-paper:
```

Comment:
Can we just call it `yellow-paper`? I like the consistency with the repo name and the folder name...